### DISCUSSION TITLE
http://frp-bar.top/ 南宁电信2 BBQ了吗

### DISCUSSION THREAD

**Discussion by klcb2010** [natfrp/wiki#339] [水深火热]
![截屏图片](https://github.com/natfrp/wiki/assets/32628414/3ae41514-a09c-4811-b404-493c7ce14d24)

**Comment by whao123yahoo** [comment]
我的 [REDACTED] 也差不多这样了，还没用过呢。
尊敬的用户您好：
您访问的网站被机房安全管理系统拦截，有可能是以下原因造成：
1.您的网站未备案，或者原备案号被取消.
2.域名在访问黑名单中.
3.网站存在非法内容.
